### PR TITLE
Start working stage

### PR DESCRIPTION
**Pull request by joe513**
Зайди в почту.

### CODE REVIEW

File: aliexpress/models.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+
+class Price(models.Model):
+
+    price = models.IntegerField(max_length=999999)
```
Comment (inoks) [inline aliexpress/models.py]: Lets use DecimalField for price

File: aliexpress/models.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+
+class Price(models.Model):
+
```
Comment (inoks) [inline aliexpress/models.py]: It also need to contains datetime and have link to the goods (FK)

File: aliexpress/models.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    price = models.IntegerField(max_length=999999)
+
+
+class Rate(models.Model):
+
```
Comment (inoks) [inline aliexpress/models.py]: Also created_at and owner field (goods)

File: aliexpress/models.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+class Images(models.Model):
+
+    images = models.ImageField()
+
+
+class Goods(models.Model):
+
+    name = models.CharField(max_length=200)
+
+    category = models.CharField(max_length=200)
```
Comment (inoks) [inline aliexpress/models.py]: Create separate model for Category and link it via FK

File: aliexpress/models.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+    description = models.CharField(max_length=1000)
+
+    images = models.ForeignKey(Images)
```
Comment (inoks) [inline aliexpress/models.py]: Link via FK on Image model

File: aliexpress/models.py [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    images = models.ImageField()
+
+
+class Goods(models.Model):
+
+    name = models.CharField(max_length=200)
+
+    category = models.CharField(max_length=200)
+
+    description = models.CharField(max_length=1000)
```
Comment (inoks) [inline aliexpress/models.py]: TextField

File: aliexpress/models.py [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+    price = models.IntegerField(max_length=999999)
+
+
+class Rate(models.Model):
+
+    rate = models.IntegerField(max_length=100)
+
+
+class Images(models.Model):
```
Comment (inoks) [inline aliexpress/models.py]: Use singular form for model name (Image instead of Images)

File: aliexpress/models.py [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+    images = models.ImageField()
+
+
+class Goods(models.Model):
```
Comment (inoks) [inline aliexpress/models.py]: Lets use name Product instead of Goods

File: aliexpress/tests.py [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+        new_one = models.Rate(rate=9.6)
+        self.assertTrue(new_one)
+        self.assertIsNone(new_one.save())
+        return new_one
+
+    def test_pollute_Category(self):
+        new_one = models.Category(category='Books')
+        self.assertTrue(new_one)
+        self.assertIsNone(new_one.save())
+        return new_one
```
Comment (inoks) [inline aliexpress/tests.py]: You don't need to return anything in the tests. It's fine to remove return line from each tests.

File: aliexpress/models.py [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+    def __str__(self):
+        return self.name
+
+
+class Price(models.Model):
+
+    price = models.DecimalField(decimal_places=3, max_digits=9999999)
+
+    date_time = models.DateTimeField()
```
Comment (inoks) [inline aliexpress/models.py]: Please use created_at name for date time fields for consistency.

File: aliexpress/models.py [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+    category = models.CharField(max_length=100)
+
+
+class Product(models.Model):
+
+    name = models.CharField(max_length=200)
+
+    description = models.TextField()
+
```
Comment (inoks) [inline aliexpress/models.py]: You need to include link to category

File: aliexpress/tests.py [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+
+    def test_pollute_Price_model(self):
+        new_one = models.Price(price=676.50, date_time=datetime.datetime.now(), product=self.test_pollute_Product_model())
```
Comment (inoks) [inline aliexpress/tests.py]: for datetime fields always use TZ aware dates.
https://docs.djangoproject.com/en/2.0/topics/i18n/timezones/#naive-and-aware-datetime-objects

File: aliexpress/tests.py (hunk 9 above)
Comment (joe513) [reply aliexpress/tests.py]: If I remove return lines I will need to create new model instances to test current model instances. For example:

        `new_one = models.Price(price=676.50, date_time=datetime.datetime.now(), product=self.test_pollute_Product_model())`

I will forced to create a new instance:

 `new_one = models.Price(price=676.50, date_time=datetime.datetime.now(), product=models.Product(name='Gold', description="Scrooge McDuck's gold", rate=self.test_pollute_Rate_model(),
                                 category=self.test_pollute_Category_model())`

This contradicts the DRY rule.

Comment (inoks) [reply aliexpress/tests.py]: You can create DB objects for multiple tests inside TestCase `setUp()`: https://docs.djangoproject.com/en/2.0/topics/testing/overview/#writing-tests

Comment (joe513) [reply aliexpress/tests.py]: thanks, I've done this

**Pull request merged**